tackle-test-generator-unit/test/java/org/konveyor/tackle/testgen/TestUtils.java

```diff
@@ -917,12 +917,12 @@ public IntegrationAppUnderTest(String appName,
             super(appName, appClasspath, appPath, appName + "_" + Constants.CTD_OUTFILE_SUFFIX);
             this.testSeqFilename = testSeqFilename;
             this.targetClassList = targetClassList;
-            
+
             if (this.testSeqFilename == null) {
                 this.testSeqFilename = appName + "_" + EvoSuiteTestGenerator.class.getSimpleName() + "_" +
-                    Constants.INITIALIZER_OUTPUT_FILE_NAME_SUFFIX + "," +
-                    appName + "_" + RandoopTestGenerator.class.getSimpleName() + "_" +
                     Constants.INITIALIZER_OUTPUT_FILE_NAME_SUFFIX;
+//                    + "," + appName + "_" + RandoopTestGenerator.class.getSimpleName() + "_" +
```

Comment:
The flaky tests are in the cli regression, not in the core regression. Please revert this change.

Reply:
This test was also failing consistently (I tried 3-4 runs); I can revert it if you'd prefer that

Reply:
Reverted